src/components/svg-canvas-graph.tsx

```diff
@@ -252,6 +327,18 @@ const SvgCanvas = () => {
 
     const SingleColor = singleColor.component;
 
+    const getPolylinesPath = (p: Polyline): [number, number, number, number] => {
+        if (p.a === 0) {
+            return [-10000, 10000, -p.c / p.b, -p.c / p.b];
```

Comment:
If the station and/or user are looking on is beyond 10000, what will happen? Wil the line be missing?